### PR TITLE
fix early crash on python3 + PyQt5

### PR DESCRIPTION
**Pull request by sthalik**
Must pass "bytes" data type to QByteArray in PyQt5.

### CODE REVIEW

File: cola/compat.py [hunk 1]
```diff
@@ -14,7 +14,12 @@
 
 try:
     # pylint: disable=bytes-builtin
-    bstr = bytes
+    bytes()
```
Comment (davvid) [inline cola/compat.py]: this should work sans the `()`.... could even do `b = bytes`

File: cola/compat.py [hunk 2]
```diff
@@ -14,7 +14,12 @@
...
+    if PY3:
+        def bstr(x):
+            return bytes(x, encoding='latin1')
```
Comment (davvid) [inline cola/compat.py]: this is starting to look suspiciously like `core.encode()`.  Shouldn't it be `utf8`?

File: cola/widgets/standard.py [hunk 3]
```diff
@@ -128,9 +129,12 @@ def apply_state(self, state):
         if windowstate is None:
             result = False
         else:
+            windowstate = str(windowstate)
+            if qtpy.PYQT5:
+                windowstate = str.encode(windowstate, encoding='us-ascii')
```
Comment (davvid) [inline cola/widgets/standard.py]: this should probably use `core.encode(...)`

**Comment by davvid** [comment]
Notes aside, i think I'll take this PR as-is and see what I can tweak on my end.  Thanks for the fix!

File: cola/compat.py (hunk 2 above)
Comment (sthalik) [inline cola/compat.py]: Is it for human-readable strings? If so, definitely.

**Pull request merged**